lib/rbi/rbs/type_translator.rb

```diff
@@ -141,6 +149,28 @@ def translate_function(type)
           proc.returns(translate(type.return_type))
           proc
         end
+
+        #: (String type_name) -> String
+        def translate_t_generic_type(type_name)
+          case type_name
+          when "Array"
+            "T::Array"
+          when "Class"
+            "T::Class"
+          when "Enumerable"
+            "T::Enumerable"
+          when "Enumerator"
+            "T::Enumerator"
+          when "Hash"
+            "T::Hash"
+          when "Set"
+            "T::Set"
+          when "Range"
+            "T::Range"
+          else
+            type_name
+          end
```

Comment:
I think we are missing `T::Enumerator::Chain` and `T::Enumerator::Lazy` here: https://github.com/sorbet/sorbet/blob/master/rbi/sorbet/t.rbi#L323-L376

Reply:
Also, what if the name was fully qualified, like `::Array[String]`?